server/tests/api/test_elections.py

```diff
@@ -385,11 +385,14 @@ def test_list_organizations_not_authorized(
     jurisdiction_ids: List[str],  # pylint: disable=unused-argument
 ):
     aa_user = User.query.filter_by(email=DEFAULT_AA_EMAIL).one()
+    db_session.expunge(aa_user)  # pylint: disable=no-member
```

Comment:
If someone can enlighten me as to how I can get mypy to learn that `scoped_session` does indeed have an `expunge` member, that would be lovely.